src/vision/status_quo/alan_started_trusting_the_rust_compiler.md

```diff
@@ -0,0 +1,59 @@
+# 😱 Status quo stories: Alan started trusting the Rust compiler
+
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+## The story
+### Trust the compiler
+Alan has a a lot of experience in _GC'd language_, but in the meantime has created some successful projects in Rust.
```

Comment:
What language? Let's be specific -- if it doesn't matter, pick one, and then write in the FAQ how it could have been any one.